assets/src/blocks/amp-story-text/get-tag-name.js

```diff
@@ -25,5 +38,15 @@ export default function( attributes ) {
 		return 'h2';
 	}
 
+	const textLength = count( attributes.content, wordCountType, {} );
+
+	if ( H1_TEXT_LENGTH >= textLength ) {
```

Comment:
I'm assuming it's intentional that the font size gets priority over the word count -- maybe it would be good to add a comment about this so that no one would have the idea to merge the word count conditions with the previous conditions? Not a real issue though.

Reply:
Intentional in the way that a huge font size is probably a better signal of a heading than the word count.